### DISCUSSION TITLE
Ender 3v2/mainboard 4.2.7/SpriteExtruder+CR-Touch

### DISCUSSION THREAD

**Discussion by BoomShanka99** [mriscoc/Ender3V2S1#1229] [Q&A] [answered]
Hello guys,

Im having strange problem. I have updated my Ender3v2 with Sprite+CR-touch. And with original firmware from creality its nightmare so I wanted to try this Pro soft which is awesome! But not working for me. I cant even get started with homing and it goes to Kill status and I have to reset the printer by power shutdown. I have found end stop status option and there I see I have z_min in TRIGGERED  status. (Also filament detector but that one I don use so I unchecked the option in filament settings so that one should be ok.) 

And Im guessing that z_min in triggered mode will block the start of Home comand.. 

Im using this firmware version --Ender3V2-427-BLTUBL-T13-MPC-20231215--- which I found on one of the releases (and should be most relevant to my HW) as Im not yet able to compile firmware.

I have tested all  conections and also i have completely dismantled and checked all features.

With original frimaware im able to print and all featurees works but I have bad print results (layer inconsistency on rounded objects--front of benchy) 

Tested multiple filaments fresh unpacked.

![WhatsApp Image 2024-01-25 at 10 24 07_1ac5e850](https://github.com/mriscoc/Ender3V2S1/assets/157694459/61cc9fc9-3e8b-43e7-aedf-c34b998c69f7)

I have checked al videos with configs and tested everything suggested so as a last resort Im on the thought that firmware I have is not compatible with my setup somehow (even if its absolutely common).

Can u please share some thoughts,advice or link to firmware which can solve my prob?

Thanks a lot in advance.

**Comment by Koky05** [comment]
Try look here:
[FB Group: MRiscoC's Professional Firmware Group for Creality Ender 3D printers](https://www.facebook.com/reel/1119899225570687)

**Comment by mriscoc** [comment, marked as answer]
> I cant even get started with homing

Are you sure that your board is a 4.2.7 version? If it is, check in the Control/Info menu if you actually have the firmware in corresponding to your board version.

**Comment by BoomShanka99** [comment]
Can be closed. Working perfect. Im just selftrusting idiot. Thanks a lot for a grat job!

**Reply by mriscoc** [reply]
Thank you for your contribution!

**Reply by GeneralSnowball** [reply]
do you still have this firmware can i get a copy

**Comment by BoomShanka99** [comment]
Thank u for the great job!

Best regard Ivo Melichar Azimut Telecom [~~Deleted for privacy~~]
